packages/cli/src/util/createServer.ts

```diff
@@ -23,6 +23,8 @@ export async function createMultiProcessPrism(options: CreatePrismOptions) {
     if (worker.process.stdout) {
       pipeOutputToSignale(worker.process.stdout);
     }
+
```

Comment:
If it’s an async function doesn’t it always return a promise? You could return true or just return nothing instead of this?

Reply:
True that, I'm just used to the old school. Fixed. f508a02